plugins/inputs/opcua/README.md

```diff
@@ -46,6 +46,12 @@ Plugin minimum tested version: 1.16
   ## Password. Required for auth_method = "UserName"
   # password = ""
   #
+  ## Option to select the metric timestamp to use. Valid options are:
+	##     "gather" -- uses the time of receiving the data in telegraf
+	##     "server" -- uses the timestamp provided by the server
```

Comment:
shouldn't gather and server be the same thing?

Reply:
"gather" is what is the current behaviour I think..